src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -101,23 +177,31 @@ def get(self, request: Request, project, rule) -> Response:
 
         return Response(serialized_rule)
 
+    @extend_schema(
+        operation_id="Update an Issue Alert Rule",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            GlobalParams.PROJECT_SLUG,
+            IssueAlertParams.ISSUE_RULE_ID,
+        ],
+        request=ProjectRuleDetailsPutSerializer,
+        responses={
+            200: RuleSerializer,
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=IssueAlertExamples.UPDATE_PROJECT_RULE,
+    )
     @transaction_start("ProjectRuleDetailsEndpoint")
     def put(self, request: Request, project, rule) -> Response:
         """
-        Update a rule
-
-        Update various attributes for the given rule.
-
-            {method} {path}
-            {{
-              "name": "My rule name",
-              "conditions": [],
-              "filters": [],
-              "actions": [],
-              "actionMatch": "all",
-              "filterMatch": "all"
-            }}
+        Updates an individual issue alert rule's attributes. Only the attributes submitted are modified.
```

Comment:
nit: `Updates an individual issue alert rule's attributes` is kind of duplicating the message from `Only the attributes submitted are modified`. Maybe just `Updates an issue alert rule. Only the ...`